libs/auth-api-lib/seeders/data/scope-application-system-admin.ts

```diff
@@ -1,6 +1,18 @@
 import { compose, createScope } from './helpers'
 
 export const up = compose(
+  createScope({
```

Comment:
Please contact and keep @jonarnarbriem in the loop to verify the configuration in each deployed environment as the seed script might be buggy, it has not been maintained for a while.